sllm/serve/routers/roundrobin_router.py

```diff
@@ -304,6 +304,12 @@ async def _start_instance(self, instance_id):
         return instance_id
 
     async def _stop_instance(self, instance_id: Optional[str] = None):
+        # check if the instance has not started
+        while (
```

Comment:
I'm not sure if this check can fully guarantee that the given `instance_id` is in `ready_instances`. Specifically, should this logic involve `starting_instances` at all?

Reply:
The error happens when all instances are still in `starting_instance` waiting for resource allocation, so `ready_instances` is empty when `instance = self.ready_instances.popitem()` is executed below. I think checking if `len(self.ready_instances) > 0` would work (without involving the `starting instance`). It can wait until some instances are added to ready, or simply return.

Reply:
OK, LGTM.